helm/ingress-nginx/values.yaml

```diff
@@ -757,10 +801,7 @@ controller:
       tolerations: []
       # -- Labels to be added to patch job resources
       labels: {}
-      securityContext:
-        runAsNonRoot: true
-        runAsUser: 2000
-        fsGroup: 2000
+      securityContext: {}
```

Comment:
the API for the patchWebhookJob is quite confusing. this is the podSecurityContext, right? Maybe would it be worth to change it (or put a comment above?)

Reply:
This is the pod-level security context being used for both jobs, certificate creation and webhook patching. I only changed the values, it already was undocumented like that in upstream.

Reply:
And yes, I agree, the values around the admission webhook jobs are shitty. I'd not change anything there and only refactor them in the future.

Reply:
I see. I'd add the comment though.

Reply:
kk, gonna do so here and in upstream.